packages-web/badge/.gitignore

```diff
@@ -0,0 +1,18 @@
+/tests/TestProjects/**/.classpath
+/tests/TestProjects/**/.project
+/tests/TestProjects/**/javascriptsource
+/tests/TestProjects/**/javasource
+/tests/TestProjects/**/resources
+/tests/TestProjects/**/userlib
+
+/tests/TestProjects/**/widgets/*
+!/tests/TestProjects/**/widgets/Badge.mpk
+!/tests/TestProjects/**/widgets/com.mendix.widget.web.Badge.mpk
+
+/tests/TestProjects/Mendix8/theme/styles/native
+/tests/TestProjects/Mendix8/theme/styles/web/sass
+/tests/TestProjects/Mendix8/theme/*.*
+!/tests/TestProjects/Mendix8/theme/components.json
```

Comment:
Why denying the ignore?

Reply:
What do you mean?

Reply:
theme should not be committed at all.

Reply:
I think I get what you mean. The majority of the files need to be ignored, so it's easier to ignore everything and exclude a few.

Reply:
yeap